swc/tests/versions_test.bzl

```diff
@@ -7,7 +7,10 @@ load("//swc/private:versions.bzl", "TOOL_VERSIONS")
 
 def _smoke_test_impl(ctx):
     env = unittest.begin(ctx)
-    asserts.equals(env, "v1.3.21", TOOL_VERSIONS.keys()[0])
+
+    # TODO: assert instead that the TOOL_VERSIONS.keys()[0] is the newest version
```

Comment:
Going to do that now? Or later?

Reply:
later; don't feel like getting into semver version parsion atm. leaving this breadcrumb here for the next time someone stumbles across this snapshot test